cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java

```diff
@@ -27,8 +29,10 @@ public abstract class CradleManager extends CradleManagerModule
 {

```

Comment:
The CradleManagerModule class looks like as redundant, maybe merge CradleManagerModule into CradleManager